packages/core/tracing/src/components/trace-viewer/SpanAttribute.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <ConfigCardItem
+    :item="{
+      type: ConfigurationSchemaType.Text,
+      key: keyValue.key,
+      label: keyValue.key,
+      value: formattedValue,
+    }"
+  >
+    <template
+      v-for="attrKey in ENTITY_LINKED_ATTRIBUTE_KEYS"
+      :key="attrKey"
+      #[attrKey]
+    >
+      <EntityLink
+        v-if="entityLink && entityLinkData"
+        allow-copy
+        :entity-link-data="entityLinkData"
+        :external-link="entityLink"
+        new-window
+      />
+    </template>
+  </ConfigCardItem>
+</template>
+
+<script setup lang="tsx">
+import { ConfigCardItem, ConfigurationSchemaType, EntityLink, type EntityLinkData } from '@kong-ui-public/entities-shared'
+import type { IKeyValue } from '@opentelemetry/otlp-transformer'
+import { computed, inject, onWatcherCleanup, ref, shallowRef, watchEffect, type PropType } from 'vue'
+import { SPAN_ATTRIBUTE_VALUE_UNKNOWN, SPAN_ATTRIBUTES, TRACE_VIEWER_CONFIG } from '../../constants'
+import type { Span, TraceViewerConfig } from '../../types'
+import { getPhaseAndPlugin, unwrapAnyValue } from '../../utils'
+
+import '@kong-ui-public/entities-shared/dist/style.css'
+
+const props = defineProps({
+  span: {
+    type: Object as PropType<Span>,
+    required: true,
+  },
+  keyValue: {
+    type: Object as PropType<IKeyValue>,
+    required: true,
+  },
+})
```

Comment:
I think we can use type style `defineProps`?

Reply:
Yes. I have replaced the object-styled `defineProps` without validators with type-styled ones.